onnx/backend/test/case/model/__init__.py

```diff
@@ -44,8 +44,8 @@ def collect_testcases():
         ('test_resnet50', 'resnet50'),
         ('test_shufflenet', 'shufflenet'),
         ('test_squeezenet', 'squeezenet'),
-        ('test_vgg16', 'vgg16'),
         ('test_vgg19', 'vgg19'),
+        ('test_zfnet', 'zfnet'),
```

Comment:
what's zfnet? :)

Reply:
Another model for image classification, won ImageNet 2013

Reply:
More details here: https://github.com/onnx/models/tree/master/zfnet

:-)

Reply:
I've never heard of it either. But LGTM.. :)

Reply:
The paper has high citation: 3000+ :-)
Let's read it.